css/alx-dark.css

```diff
@@ -45,8 +49,12 @@ body.dark hr {
 }
 
 body.dark code {
-  background-color: var(--col-bg-400) !important;
-  color: var(--col-acc-500) !important;
+  background-color: var(--col-code-bg) !important;
+  color: var(--col-code-color) !important;
```

Comment:
You'd use previous code for the background color and maintain your changes for the color property (kindly ensure you've set the value to #e55039)